### PR TITLE
Added kube-burner output to archived artifacts

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Also moved the agent definition as the previous definition made it unable to run the post step

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Tested here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/kube-burner/15/

File: Jenkinsfile [hunk 1]
```diff
@@ -328,4 +327,17 @@ pipeline {
             }
         }
     }
+    post {
+        always {
+            println 'Post Section - Always'
+            archiveArtifacts(
+                artifacts: 'workloads/kube-burner/kube-burner.out',
```
Comment (paigerube14) [inline Jenkinsfile:334]: I believe if the scaling fails in this job there won't be that file, will the allowEmptyArchive cover that or should you add in an extra check to verify the file exists

Comment (nathan-weinberg) [reply Jenkinsfile:334]: @paigerube14 the `allowEmptyArchive` should cover that :+1:

**Pull request merged**